Modules/zlibmodule.c

```diff
@@ -1876,6 +1886,44 @@ zlib_adler32_impl(PyObject *module, Py_buffer *data, unsigned int value)
     return PyLong_FromUnsignedLong(value & 0xffffffffU);
 }
 
+/*[clinic input]
+zlib.adler32_combine -> unsigned_int
+
+    adler1: unsigned_int(bitwise=True)
+        Adler-32 check value for sequence A
+
+    adler2: unsigned_int(bitwise=True)
+        Adler-32 check value for sequence B
+
+    blen: object
```

Comment:
```suggestion
    blen: object(subclass_of='&PyLong_Type')
```

Maybe good to let argument clinic check we're passed an integer?

Reply:
Oups, I totally missed this one actually! there will be a TypeError raised during conversion I think, but yes, AC could do it as well.